### PR TITLE
Added benchmark test for cluster.Flush()

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What this PR does / why we need it**:
Added benchmark test for cluster.Flush()

### CODE REVIEW

File: pkg/cache/v2/cluster_test.go [hunk 1]
```diff
@@ -171,3 +178,177 @@ func TestClusterFlush(t *testing.T) {
...
+		LbPolicy: cluster_v2.Cluster_ROUND_ROBIN,
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		cache := NewClusterCache()
+		cluster.Name = rand.String(6)
+		cache.SetApiCluster(cluster.Name, &cluster)
+
+		cache.Flush()
```
Comment (hzxuzhonghu) [inline pkg/cache/v2/cluster_test.go:351]: You didn't set the api status, actually the flush do nothing

Comment (LiZhenCheng9527) [reply pkg/cache/v2/cluster_test.go:351]: I set Cluster.ApiStatus in L229. and compared the value of api status after Flush() processing. 
Its changed from `update` to `none`, which shows that Flush did the processing.

**Comment by LiZhenCheng9527** [comment]
```console
goos: linux
goarch: amd64
pkg: kmesh.net/kmesh/pkg/cache/v2
cpu: Intel(R) Xeon(R) Gold 6278C CPU @ 2.60GHz
BenchmarkClusterFlush-4              100        2632608350 ns/op
PASS
ok      kmesh.net/kmesh/pkg/cache/v2    263.388s
```

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**